examples/tfengine/full.yaml

```diff
@@ -18,6 +18,10 @@ data:
   ORG_ID: "123"
   BILLING_ACCOUNT: "000-000-000"
   STORAGE_LOCATION: "us-central1"
```

Comment:
btw, this storage_location is somewhat confusing and it's not clear what location it actually controls, and it's not setting the location for everything. We should consider removing this and just set the location at each individual block.

Reply:
Would renaming this to STORAGE_BUCKET_LOCATION be clearer? I think it's helpful if you want to avoid having to duplicate the field everywhere.

Reply:
Does it apply to all storage bucket locations? i.e. state bucket, audit log sink bucket, forseti buckets (currently not) and any user created buckets?

Reply:
Or we can go for a way that this is a convenient field to control everything, if users don't override the location in individual data blocks.

Reply:
state and audit log sink bucket yes, i'm not sure about the forseti one, i don't think we pass any location to it yet.

WDYT about supporting both a global field and allowing individual overrides?

Reply:
supporting both a global field and allowing individual overrides sgtm. If we can actually set on location and apply to everything, that will be fantastic, to buckets, sql, datasets, anything. So make it hard for users to set one in us-east1 and one in us-central1 like what we did in mystudies.

One place might need to look more carefully is if the same type of region or zone is supported by all those servces. i.e. the syntax for region of buckets can potentially be different for sql? or some zone is supported by bucket but not in bigquery. I guess we can start with one global attribute for one type of service to be safe.

Reply:
I originally wanted that, but the problem is not every resource is available in every region. For example, healthcare API is not available in us-east1.

Let me try to do global + local override for every resource for now

Reply:
sg

Reply:
we can do that in a separate PR.